server/custom/reports/controller.js

```diff
@@ -17,8 +17,73 @@ exports.ReportsFindAll = async function (req, res) {
     req.query.companyid = true;
     req.user = {};
     req.user.companyID = 'COMPID';
-    const result = await controller.findAll(req);
-    res.status(200).json(result);
+    var perPage = config.get('pagination.itemsPerPage');
+    var page = (req.query.page) ? req.query.page : 1;
+    var result = controller.findAllComponent(req);
+    var count = await Reports.count(result.find).exec();
+    if (req.query.populate === 'layer') {
+        var aggregation = Reports.aggregate([
+            {
+                $lookup:
+        {
+            from: 'wst_Layers',
+            localField: 'selectedLayerID',
+            foreignField: '_id',
+            as: 'layer'
+        }
```

Comment:
Please fix indentation.
Ideally, a value should start on the same line as the property:

```js
$lookup: {
    // ...
}
```